gprofiler/utils/fs.py

```diff
@@ -0,0 +1,13 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import os
+import shutil
+
+
+def atomic_copy(src: str, dst: str) -> None:
```

Comment:
This is not atomic, while two threads (and especially two processes) are running this function concurrently the file at `dst` might be corrupt.  
This is not a bug, since we use a lock when calling this function, but this is a misleading function

Reply:
I suppose `atomic` is misleading. This aims to be "atomic" at the filesystem level. I'll document and rename to `safe_copy`.